Library/Homebrew/cmd/pull.rb

```diff
@@ -189,8 +189,14 @@ def pull
               "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
               "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
             puts
-            sleep 20
-            safe_system "brew", "fetch", "--retry", "--force-bottle", f.full_name
+          end
+          unless (Pathname.pwd/".git/hooks/pre-push").exist?
```

Comment:
`Pathname.pwd` should be `HOMEBREW_REPOSITORY`. So we can run `brew pull` anywhere.